nixos/modules/services/networking/webhook.nix

```diff
@@ -0,0 +1,160 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.webhook;
+  defaultUser = "webhook";
+
+  hooksFile = pkgs.writeText "webhook-config" cfg.hooks;
+
+  # https://github.com/adnanh/webhook/blob/master/docs/Hook-Rules.md
+  # We might merge with "or" (perhaps?), but until then, allow only a single definition.
+  typeTriggerRule = types.uniq types.attrs;
+
+in {
+  options = {
+    services.webhook = {
+      enable = mkEnableOption (mdDoc ''
+        [Webhook] is a server written in Go that allows you to create HTTP endpoints (hooks),
+        which execute configured commands for any person or service that knows the URL.
+
+        [Webhook]: https://github.com/adnanh/webhook
+      '');
```

Comment:
Since this is a `mkEnableOption` the text will be wrapped in `Whether to enable {}.`. You could either adapt it to fit that format, override the `description`, or just use `mkOption`.

Reply:
I'll adapt the text to fix the `Whether to enable` prefix.

Reply:
Done in c7c7371